.travis.yml

```diff
@@ -50,21 +50,22 @@ before_install:
     - phpdbg --version 2> /dev/null || { echo 'No phpdbg'; export TASK_TESTS_COVERAGE=0; }
 
     # turn off XDebug
-    - phpenv config-rm xdebug.ini || return 0
+    - phpenv config-rm xdebug.ini || true
 
     # validate tasks configuration
-    - if [ $TASK_TESTS == 0 ] && [ $TASK_TESTS_COVERAGE == 1 ]; then travis_terminate 1; fi
+    - if ((!TASK_TESTS & TASK_TESTS_COVERAGE)); then travis_terminate 1; fi
```

Comment:
I'm not saying it won't run, I'm saying that `[` is supposed to be use for logic checks.